### PR TITLE
FY21Q2 content refresh

### PR DESCRIPTION
**Pull request by andrewconnell**
- run through all exercise steps in associated MSLearn module to validate good working order
- updated steps & fixed typos where necessary
- updated slides where necessary
- update code samples to reflect MSLearn HOL exercise refresh & updates

### CODE REVIEW

File: Demos/01-create-app/graph-tutorial/Controllers/HomeController.cs [hunk 1]
```diff
@@ -1,7 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
-using System;
+﻿using System;
```
Comment (jasonjoh) [inline Demos/01-create-app/graph-tutorial/Controllers/HomeController.cs:1]: A number of files here dropped the copyright notice.

File: Demos/01-create-app/graph-tutorial/Startup.cs [hunk 2]
```diff
@@ -1,10 +1,7 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
+﻿using Microsoft.Owin;
```
Comment (jasonjoh) [inline Demos/01-create-app/graph-tutorial/Startup.cs:1]: Copyright

File: Demos/02-add-aad-auth/graph-tutorial/Controllers/HomeController.cs [hunk 3]
```diff
@@ -1,7 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
-using System;
+﻿using System;
```
Comment (jasonjoh) [inline Demos/02-add-aad-auth/graph-tutorial/Controllers/HomeController.cs:1]: Copyright

File: Demos/03-add-msgraph/graph-tutorial/Controllers/HomeController.cs [hunk 4]
```diff
@@ -1,7 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
-using System;
+﻿using System;
```
Comment (jasonjoh) [inline Demos/03-add-msgraph/graph-tutorial/Controllers/HomeController.cs:1]: Copyright

**Review by jasonjoh — CHANGES_REQUESTED** [review]

**Comment by andrewconnell** [comment]
Sorry about the copyright issue... slipped my edits. Fixed in latest commit.

**Review by jasonjoh — APPROVED** [review]

**Comment by jasonjoh** [comment]
Thanks!

**Pull request merged**